src/tokens.ts

```diff
@@ -0,0 +1,6 @@
+export enum ProviderTokens {
+    UserService = "UserService",
+    ExampleService = "ExampleService",
+    PostgresDataSource = "PostgresDataSource",
+    UserPostgres = "UserPostgres"
```

Comment:
Oh! this reminds me about trailing commas https://eslint.org/docs/latest/rules/comma-dangle - it's in our spec somewhere and makes for cleaner diffs

Reply:
agreed